packages/keyring-controller/src/KeyringController.ts

```diff
@@ -116,6 +117,11 @@ export type KeyringControllerSignMessageAction = {
   handler: KeyringController['signMessage'];
 };
 
+export type KeyringControllerSignAuthorizationMessageAction = {
```

Comment:
I've been using this temporary type for reference:

https://github.com/MetaMask/core/blob/main/packages/transaction-controller/src/utils/eip7702.ts#L18

Happy to update as needed to be more explicit but should we omit `Message` to match the method name?

Reply:
Agree with omitting the `Message` part.

Reply:
Done!

Reply:
We still have the `Message` part in the type no?

Reply:
Sorry, fixed this now.